utils/mstflint/Makefile

```diff
@@ -0,0 +1,101 @@
+#
+# Copyright (C) 2024 Til Kaiser
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=mstflint
+PKG_VERSION:=4.26.0-1
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/Mellanox/mstflint
+PKG_MIRROR_HASH:=7d0168add5857f54e2f96edb6cc78ff309fad78c404e262f145b89c63f6db39c
+PKG_SOURCE_VERSION:=17e9b849ba57cfac1b28240a25ece300a0f74a45
+PKG_SOURCE_DATE:=2024-01-15
```

Comment:
We dont need to use the latest commit and we could stick to the stable/tagged releases, right?